tests/conftest.py

```diff
@@ -2507,3 +2512,15 @@ def measurement_dir(tmp_path):
         )
         os.mkdir(measurement_dir)
     return measurement_dir
+
+
+@pytest.fixture(scope="session", autouse=True)
+def ceph_toolbox(request):
+    """
+    This fixture initiates ceph toolbox pod for manually created deployment
```

Comment:
I thought we had this already?

Reply:
If it doesn't exist it does create it on the fly

Reply:
@vasukulkarni we didn't have it.  We do deploy cep_toolbox as part of our deployment but if the cluster was deployed manually and not via ocs-ci then no one deployed ceph toolbox pod and this fixture taking care of it on such clusters.